pylint/checkers/classes/class_checker.py

```diff
@@ -1408,14 +1408,10 @@ def _check_slots_elt(self, elt, node):
             if not isinstance(inferred, nodes.Const) or not isinstance(
                 inferred.value, str
             ):
-                self.add_message(
-                    "invalid-slots-object", args=inferred.as_string(), node=elt
-                )
+                self.add_message("invalid-slots-object", args=elt.as_string(), node=elt)
```

Comment:
```suggestion
                self.add_message("invalid-slots-object", args=elt.as_string(), node=elt, confidence=interface.INFERENCE)
```
Not related to the current fix but we may as well fix that. Maybe another commit so we can rebase later ?